blobfuse2-nightly.yaml

```diff
@@ -888,4 +888,85 @@ jobs:
               sudo rm -rf [0-9]
             displayName: 'Clean Agent Directories'
             condition: always() 
-  
\ No newline at end of file
+  - job: Set_8
```

Comment:
This step tries to create a package and then install it on a container. Shall we do e2e tests inside the container just to make sure other functionalities are also working. If so we shall change the container name as other pipeline jobs will also be using the same (for regular ubn-18 jobs)

Reply:
We can create a separate issue to run E2E tests since that might be more work - figuring out how to get the env variables passed through and everything